Packages/Sites/Sfi.Kateheo/Resources/Private/Templates/NodeTypes/Tags.html

```diff
@@ -5,7 +5,7 @@
 
 <f:section name="Content">
 	<div class="Tags">
-		<h1 class="Tag-Name">{neos:contentElement.editable(property: 'title', tag: 'span')}</h1>
+		<h1 class="Tag-Name">Список материалов по тегу: {neos:contentElement.editable(property: 'title', tag: 'span')}</h1>
```

Comment:
I think it's because of copy&past from this (I use category NodeType and template for Authors and then for Tags):
https://github.com/psmb/KateheoDistr/blob/9b5c980f07027cfcefc74d2dee698c7563587fbf/Packages/Sites/Sfi.Kateheo/Resources/Private/Templates/NodeTypes/Category.html

As I understand - "editable" allows to edit Tag, Author title in back-end. I think, you aware of this :) But I don't know - is it necessary to edit title this way?